packages/context/src/context.ts

```diff
@@ -2,6 +2,8 @@ import { Slot } from "./slot";
 export { Slot }
 export const { bind, noContext } = Slot;
 
+declare function setTimeout(callback: (...args: any[]) => void, ms: number, ...args: any[]): any;
```

Comment:
```suggestion
declare function setTimeout(callback: (...args: any[]) => any, ms?: number, ...args: any[]): any;
```
Any objection to these tweaks?